SConscript

```diff
@@ -463,7 +463,7 @@ def buildLibs(projectDir):
     libs_dict = {
         '00std': [                                                              ],
         '01bsp': [                                                      'libbsp'],
-        '02drv': [                                         'libdrivers','libbsp'],
```

Comment:
Otherwise it failed in linking the project together complaining that it could not find scheduler_init and scheduler_start. See below:

scons board=OpenMote-CC2538 toolchain=armgcc drv_sensors
scons: Reading SConscript files ...

 ___                 _ _ _  ___  _ _ 
| . | ___  ___ ._ _ | | | |/ __>| \ |
| | || . \/ ._>| ' || | | |\__ \|   |
`___'|  _/\___.|_|_||__/_/ <___/|_\_|
     |_|                  openwsn.org

scons: done reading SConscript files.
scons: Building targets ...
Linking   projects/common/02drv_sensors_prog
projects/common/02drv_sensors/02drv_sensors.o: In function `mote_main':
/home/pere/Escritorio/OpenMote/OpenWSN/openwsn-fw//projects/common/02drv_sensors/02drv_sensors.c:38: undefined reference to `scheduler_init'
/home/pere/Escritorio/OpenMote/OpenWSN/openwsn-fw//projects/common/02drv_sensors/02drv_sensors.c:59: undefined reference to `scheduler_start'
collect2: error: ld returned 1 exit status
scons: *** [projects/common/02drv_sensors_prog] Error 1
scons: building terminated because of errors.